packages/scaffolding-cli/templates/src/csr/package.json

```diff
@@ -115,8 +119,8 @@
     "test:cypress:run": "npm run test:cypress:build && node_modules/.bin/cypress run --spec \"**/*.cy.js\"",
     "test:cypress:axe:run": "npm run test:cypress:run -- --spec \"**/*.test.axe.cy.js\"",
     "test:cypress:open": "npm run test:cypress:build && node_modules/.bin/cypress open",
-    "test:cypress": "env CI=true node_modules/.bin/start-server-and-test start https://$HOST:$PORT test:cypress:run",
-    "test:cypress:axe": "env CI=true node_modules/.bin/start-server-and-test start https://$HOST:$PORT test:cypress:axe:run",
+    "test:cypress": "env CI=true node_modules/.bin/start-server-and-test start http://$HOST:$PORT test:cypress:run",
+    "test:cypress:axe": "env CI=true node_modules/.bin/start-server-and-test start http://$HOST:$PORT test:cypress:axe:run",
```

Comment:
not sure how this will work in CSR. do they run a build script to generate the statically compiled site and point to it from that instance of Express?

Reply:
webpack dev starts the process :) checkout `npm start` which points to the script/start which serves the app locally.